### PR TITLE
Typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
Run [codespell](https://github.com/codespell-project/codespell) in CI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -438,7 +438,7 @@ Full changelog @  https://github.com/drwetter/testssl.sh/commits/2.2/testssl.sh
 * local hostname in output
 
 1.31 (Halloween Release)
-* bugfix: SSLv2 was kind of borken
+* bugfix: SSLv2 was kind of broken
```
Comment (drwetter) [inline CHANGELOG.md]: this is deliberate ;-)

**Review by drwetter — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (DimitriPapadopoulos) [reply CHANGELOG.md]: Ah, will fix back to _borken_ then :smile:

**Comment by drwetter** [comment]
Cool thing! Thanks a lot!

Mind to backport that to 3.0?

**Comment by DimitriPapadopoulos** [comment]
i can try.

**Pull request merged**